### PR TITLE
add log compare test for blinky example

### PR DESCRIPTION
**Pull request by jamesbeyond**
Add a log compare test for this example for enable example CI testing

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+Examples are tested using tool [htrun](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-host-tests) and templated print log. 
+
+To run the test, use following command after you build the example:
+```
+mbedhtrun -d D: -p COM4 -m K64F -f .\BUILD\K64F\GCC_ARM\blinky.bin --compare-log tests\blinky.log
+```
+
+
+More details about `htrun` are [here](https://github.com/ARMmbed/htrun#testing-mbed-os-examples).
```
Comment (0xc0170) [inline tests/README.md]: The link is not working anymore (htrun moved)

**Comment by 0xc0170** [comment]
Question: how do I maintain this log (generate a new one, just need to edit it as I understand right) - any change to example needs log update.

File: tests/README.md (hunk 1 above)
Comment (jamesbeyond) [reply tests/README.md]: This link is fixed.

**Comment by jamesbeyond** [comment]
> Question: how do I maintain this log (generate a new one, just need to edit it as I understand right) - any change to example needs log update.

the detailed instruction is in that fixed link, the log will like to be manually maintained, they just standard RegEx. if any one change the example the log will need to be updated.

**Comment by OPpuolitaival** [comment]
I am thinking that this does not test at all that the blinking happens. What you think about adding "led on" and "led off" prints to the file and check that at least couple of those are there? Then we know at least that board is not in stuck. For sure we don't know if led is on really but at least logic works.

**Comment by 0xc0170** [comment]
> I am thinking that this does not test at all that the blinking happens. What you think about adding "led on" and "led off" prints to the file and check that at least couple of those are there? Then we know at least that board is not in stuck. For sure we don't know if led is on really but at least logic works.

@jamesbeyond please review, and send separate PR

**Comment by jamesbeyond** [comment]
> I am thinking that this does not test at all that the blinking happens. What you think about adding "led on" and "led off" prints to the file and check that at least couple of those are there? Then we know at least that board is not in stuck. For sure we don't know if led is on really but at least logic works.

I think that wouldn't be necessary. printing `led on`/`led off` are not equivalent to the LED is physically on or off. that will require manual checks. also the on and off logic is fairly simple. in this test we just want to make sure the example wasn't crash/broken badly from beginning

**Pull request merged**